app/controllers/api/v1/memes_controller.rb

```diff
@@ -1,8 +1,18 @@
 class Api::V1::MemesController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
 
   def index
     memes = Meme.all
     render json: memes
   end
 
+  def create
+    meme = Meme.new(user: current_user, title: params[:title], imageUrl: params[:imageUrl], description: params[:description])
```

Comment:
Please break up long lines like this to make them more readable: 
```suggestion
    meme = Meme.new(
       user: current_user, 
       title: params[:title], 
       imageUrl: params[:imageUrl], 
       description: params[:description]
    )
```